### PR TITLE
document menu configuration options

### PR DESCRIPTION
**Pull request by dbu**
fix https://github.com/symfony-cmf/MenuBundle/issues/216

### CODE REVIEW

**Comment by dbu** [comment]
fixed formatting error, hopefully its green now

**Comment by WouterJ** [comment]
IF the options are new in 1.2, they should get a `.. versionadded:: 1.2` directive

**Comment by dbu** [comment]
content_url_generator is new. allow_empty_items existed since 1.0 but
was never documented.

**Comment by dbu** [comment]
updated

File: bundles/menu/configuration.rst [hunk 1]
```diff
@@ -180,6 +180,26 @@ admin_recursive_breadcrumbs
 When editing a node, this setting will cause the Sonata admin breadcrumb to
 include ancestors of the node being edited.
 
+content_url_generator
+~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 1.2
+    This option was added in version 1.2.0. In older versions, this option is
```
Comment (WouterJ) [inline bundles/menu/configuration.rst]: The convention is: "was introduced in .... Prior to 1.2, this option is not available and the default service `router` was hardcoded."

File: bundles/menu/menu_factory.rst [hunk 2]
```diff
@@ -44,14 +52,24 @@ of the three URL generation techniques to use.
...
 * **route**: Generate a URL using the route named by the ``route`` option
   and the parameters provided by the ``routeParameters`` option.
-* **content**: Generate a URL by passing the ``RouteReferrersInterface``
-  instance provided by the ``content`` option to the router using any
-  parameters provided by the ``routeParameters`` option.
+* **content**: Generate a URL by passing the value of the ``content`` option to
+  the content URL generator, using any parameters provided by the
+  ``routeParameters`` option.
+
+Menu nodes without URL
```
Comment (WouterJ) [inline bundles/menu/menu_factory.rst]: "Menu Nodes without URL"

**Comment by dbu** [comment]
sorry, i hope one day i will remember to use the proper convention for was introduced without you having to remind me...

updated

**Comment by WouterJ** [comment]
:+1:

**Comment by dbu** [comment]
cool. build is green. merge?

**Pull request merged**